readme.md

```diff
@@ -63,10 +63,11 @@ Outputs the Node.js compact build of `input.js` into `dist/index.js`.
   -e, --external [mod]     Skip bundling 'mod'. Can be used many times
   -q, --quiet              Disable build summaries / non-error outputs
   -w, --watch              Start a watched build
+  -t, --transpile-only     Use transpileOnly option with the ts-loader
   --v8-cache               Emit a build using the v8 compile cache
   --license [file]         Adds a file containing licensing information to the output
   --stats-out [file]       Emit webpack stats as json to the specified output file
-  --target [es2015|es2020] Select ecmascript target to use for output
+  --target [es2015|es2020] ECMAScript target to use for output (default: es2015)
```

Comment:
```suggestion
  --target [es2015|es2020] ECMAScript target to use for output (default: es2015) [Learn more](https://webpack.js.org/configuration/target)
```

Reply:
Oh, didn't spot that earlier, MD link will not work inside code block, I will propose a small change.

<img width="635" alt="Screenshot 2021-05-02 115929" src="https://user-images.githubusercontent.com/719641/116809363-f3688480-ab3d-11eb-9d28-f7f2ff6a1eb9.png">